util/Version.h

```diff
@@ -1,15 +1,41 @@
 #ifndef _Version_h_
 #define _Version_h_
 
+/** @file
+ * @brief  Declares free functions to access the application and dependencies
+ *      versions.
+ */
+
 #include <string>
 #include <map>
 
 #include "Export.h"
 
+/** @brief  Returns the version string of FreeOrion.
+ *
+ * @return  The version string with the format
+ *      <em>VERSION BRANCH_NAME [build DATE.SHORT_COMMIT_HASH] BUILD_SYSTEM</em>
```

Comment:
might want to give an example of what "VERSION" would look like...
might want to clarify that it's a git commit hash.
might want to give example(s) of what BUILD_SYSTEM would look like
or, could just give an example of a full version string.